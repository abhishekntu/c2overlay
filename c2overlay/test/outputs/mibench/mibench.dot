digraph graphname {
N10 [color="black", label="O0_N17_N10"];
N8 [color="black", label="N6N9N8_add_N6_mul_N9_add_N8_N8"];
N9 [color="black", label="N7N16_add_N7_mul_N16_N9"];
N1 [color="black", label="I0_N1_N1"];
N2 [color="black", label="I1_N2_N2"];
N3 [color="black", label="I2_N3_N3"];
N4 [color="black", label="N12_mul_Imm_9_N12_N4"];
N5 [color="black", label="N13N15N14_add_Imm_1_N13_mul_N15_add_N14_N5"];
N6 [color="black", label="N10N5_mul_Imm_2_N10_add_N5_N6"];
N7 [color="black", label="N11N4_mul_Imm_6_N11_add_Imm_43_N4_N7"];
N8 -> N5;
N9 -> N8;
N1 -> N9;
N1 -> N6;
N2 -> N8;
N2 -> N6;
N3 -> N4;
N3 -> N7;
N3 -> N5;
N4 -> N5;
N5 -> N10;
N6 -> N8;
N7 -> N9;
N7 -> N8;
}
