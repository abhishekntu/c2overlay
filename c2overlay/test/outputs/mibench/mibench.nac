procedure foo(in s32 x,in s32 y,in s32 z,out s32 D_2407)
{
  localvar s32 D_2406;
  localvar s32 D_2408;
  localvar s32 D_2409;
  localvar s32 D_2410;
  localvar s32 D_2411;
  localvar s32 D_2412;
  localvar s32 D_2413;
  localvar s32 D_2414;
  localvar s32 D_2415;
  localvar s32 D_2416;
  localvar s32 D_2417;
  localvar s32 temp;
L0005:
  D_2406 <= mul z,6;
  temp <= add D_2406,43;
  D_2408 <= mul y,2;
  D_2409 <= add D_2408,x;
  D_2410 <= add D_2409,temp;
  D_2411 <= mul D_2410,x;
  D_2412 <= add y,temp;
  D_2413 <= mul D_2412,y;
  D_2414 <= add D_2411,D_2413;
  D_2415 <= mul z,9;
  D_2416 <= add D_2415,1;
  D_2417 <= mul D_2416,z;
  D_2407 <= add D_2414,D_2417;
}
