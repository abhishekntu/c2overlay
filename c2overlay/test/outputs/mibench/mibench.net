.input N1
pinlist: N1
    
.input N2
pinlist: N2
    
.input N3
pinlist: N3
    
.output out:N11
pinlist: N11
    
.clb N10_blk
pinlist: N6 N1 open open N10 open open open open 
subblock: N10_blk 0 1 open open 4 open open open open

.clb N8_blk
pinlist: N7 N9 N10 open N11 open open open open 
subblock: N8_blk 0 1 2 open 4 open open open open
    
.clb N9_blk
pinlist: N5 N6 N2 open N9 open open open open 
subblock: N9_blk 0 1 2 open 4 open open open open
    
.clb N4_blk
pinlist: N3 open open open N4 open open open open 
subblock: N4_blk 0 open open open 4 open open open open

.clb N5_blk
pinlist: N1 N2 open open N5 open open open open 
subblock: N5_blk 0 1 open open 4 open open open open

.clb N6_blk
pinlist: N3 open open open N6 open open open open 
subblock: N6_blk 0 open open open 4 open open open open

.clb N7_blk
pinlist: N4 N3 open open N7 open open open open 
subblock: N7_blk 0 1 open open 4 open open open open

