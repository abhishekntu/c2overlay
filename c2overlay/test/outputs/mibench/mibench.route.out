Array size: 5 x 5 logic blocks.

Routing:

Net 0 (N1)

SOURCE (0,2)  Pad: 1  
  OPIN (0,2)  Pad: 1  
 CHANY (0,2)  Track: 1  
 CHANX (1,1)  Track: 0  
 CHANX (2,1)  Track: 0  
  IPIN (2,2)  Pin: 2  
  SINK (2,2)  Class: 0  
 CHANX (1,1)  Track: 0  
  IPIN (1,1)  Pin: 0  
  SINK (1,1)  Class: 0  


Net 1 (N2)

SOURCE (2,0)  Pad: 1  
  OPIN (2,0)  Pad: 1  
 CHANX (2,0)  Track: 1  
  IPIN (2,1)  Pin: 2  
  SINK (2,1)  Class: 0  
 CHANX (2,0)  Track: 1  
 CHANY (1,1)  Track: 0  
 CHANY (1,2)  Track: 0  
  IPIN (2,2)  Pin: 3  
  SINK (2,2)  Class: 0  


Net 2 (N3)

SOURCE (5,0)  Pad: 1  
  OPIN (5,0)  Pad: 1  
 CHANX (5,0)  Track: 1  
  IPIN (5,1)  Pin: 2  
  SINK (5,1)  Class: 0  
 CHANX (5,0)  Track: 1  
 CHANX (4,0)  Track: 1  
  IPIN (4,1)  Pin: 2  
  SINK (4,1)  Class: 0  
 CHANX (4,0)  Track: 1  
 CHANX (3,0)  Track: 1  
  IPIN (3,1)  Pin: 2  
  SINK (3,1)  Class: 0  


Net 3 (N8)

SOURCE (2,1)  Class: 1  
  OPIN (2,1)  Pin: 5  
 CHANY (2,1)  Track: 1  
 CHANX (3,0)  Track: 0  
 CHANY (3,1)  Track: 0  
  IPIN (4,1)  Pin: 3  
  SINK (4,1)  Class: 0  


Net 4 (N9)

SOURCE (1,1)  Class: 1  
  OPIN (1,1)  Pin: 5  
 CHANY (1,1)  Track: 1  
  IPIN (2,1)  Pin: 3  
  SINK (2,1)  Class: 0  


Net 5 (N4)

SOURCE (5,1)  Class: 1  
  OPIN (5,1)  Pin: 7  
 CHANY (4,1)  Track: 1  
  IPIN (4,1)  Pin: 1  
  SINK (4,1)  Class: 0  


Net 6 (N10)

SOURCE (4,1)  Class: 1  
  OPIN (4,1)  Pin: 6  
 CHANX (4,0)  Track: 0  
  IPIN (4,0)  Pad: 0  
  SINK (4,0)  Pad: 0  


Net 7 (N6)

SOURCE (2,2)  Class: 1  
  OPIN (2,2)  Pin: 6  
 CHANX (2,1)  Track: 1  
  IPIN (2,1)  Pin: 0  
  SINK (2,1)  Class: 0  


Net 8 (N7)

SOURCE (3,1)  Class: 1  
  OPIN (3,1)  Pin: 7  
 CHANY (2,1)  Track: 0  
  IPIN (2,1)  Pin: 1  
  SINK (2,1)  Class: 0  
 CHANY (2,1)  Track: 0  
 CHANY (2,2)  Track: 0  
 CHANX (2,2)  Track: 1  
 CHANY (1,2)  Track: 1  
 CHANX (1,1)  Track: 1  
 CHANY (0,1)  Track: 1  
  IPIN (1,1)  Pin: 3  
  SINK (1,1)  Class: 0  
